### DISCUSSION TITLE
How to use Min/Max function with MathExpressionConverter?

### DISCUSSION THREAD

**Discussion by CodingOctocat** [CommunityToolkit/Maui#2384] [Q&A] [answered]
I noticed that the documentation supports a variety of operators, but there are no examples, I don't know how to use these operators, the examples are only simple ones such as add, subtract, multiply, divide, how do I use them for functions such as min, round?

**Comment by stephenquan** [comment]
Some references:
 - MathExpressionConverterTest.cs shows `min()` usage: https://github.com/CommunityToolkit/Maui/blob/main/src/CommunityToolkit.Maui.UnitTests/Converters/MathExpressionConverterTests.cs
 - MathExpressoin.shared.cs is the implementation: https://github.com/CommunityToolkit/Maui/blob/main/src/CommunityToolkit.Maui/Converters/MathExpressionConverter/MathExpression.shared.cs

You see that they're both thin wrappers of [Math.Min](https://learn.microsoft.com/en-us/dotnet/api/system.math.min?view=net-9.0) and [Math.Round](https://learn.microsoft.com/en-us/dotnet/api/system.math.round?view=net-9.0) respectively.

**Reply by CodingOctocat** [reply]
I tried the following code, but an Invalid exception occurs. `<Label Text="{Binding MyValue, Converter={StaticResource MathExpressionConverter}, ConverterParameter='min(x/2,0)'}" />`

**Reply by stephenquan** [reply, marked as answer]
That code snippet looks correct. Perhaps the error is in code you haven't shared?

**Reply by stephenquan** [reply]
Just something random, what happens if you add spaces, e.g. `min( x/2 , 0)` ?

**Reply by CodingOctocat** [reply]
@stephenquan You're right, there happened to be some errors in the binding.
